python/lsst/afw/geom/ellipses/__init__.py

```diff
@@ -23,23 +23,20 @@
 """lsst.afw.geom.ellipses
 """
 from __future__ import absolute_import
-from builtins import object
 from .ellipsesLib import *
-import weakref
-import numpy
 
-Separable = {
-    (Distortion, DeterminantRadius):SeparableDistortionDeterminantRadius,
-    (Distortion, TraceRadius):SeparableDistortionTraceRadius,
-    (Distortion, LogDeterminantRadius):SeparableDistortionLogDeterminantRadius,
-    (Distortion, LogTraceRadius):SeparableDistortionLogTraceRadius,
-    (ConformalShear, DeterminantRadius):SeparableConformalShearDeterminantRadius,
-    (ConformalShear, TraceRadius):SeparableConformalShearTraceRadius,
-    (ConformalShear, LogDeterminantRadius):SeparableConformalShearLogDeterminantRadius,
-    (ConformalShear, LogTraceRadius):SeparableConformalShearLogTraceRadius
-}
+#Separable = {
+#    (Distortion, DeterminantRadius):SeparableDistortionDeterminantRadius,
+#    (Distortion, TraceRadius):SeparableDistortionTraceRadius,
+#    (Distortion, LogDeterminantRadius):SeparableDistortionLogDeterminantRadius,
+#    (Distortion, LogTraceRadius):SeparableDistortionLogTraceRadius,
+#    (ConformalShear, DeterminantRadius):SeparableConformalShearDeterminantRadius,
+#    (ConformalShear, TraceRadius):SeparableConformalShearTraceRadius,
+#    (ConformalShear, LogDeterminantRadius):SeparableConformalShearLogDeterminantRadius,
+#    (ConformalShear, LogTraceRadius):SeparableConformalShearLogTraceRadius
+#}
```

Comment:
Ok. But I think that you'll find as you start wrapping that you'll be doing a lot of commenting and uncommenting code that is not yet wrapped and it would be quite burdensome to always add extra comments when doing this.